admin/howto/control-user-server.md

```diff
@@ -39,6 +39,22 @@ Accessing a user's server is useful when trying to debug or reproduce an issue t
    you are taking over their session, and using this feature mostly to understand what the user was trying to do, rather than to make major
    changes to their code or notebook outputs.
    :::
+   
+### Access user logs
+
+The logs for each user server are located in their home directory, at `~/.jupyter-server-log.txt`.
```

Comment:
This only works for images built with repo2docker, so pangeo, utoronto, etc won't have this.

Reply:
@choldgraf let's just say 'this only works for repo2docker built environments that do not use a custom Dockerfike' for now?